pokemongo_bot/cell_workers/utils.py

```diff
@@ -97,12 +98,12 @@ def i2f(int):
 
 
 def print_green(message):
-    print(u'\033[92m' + message.decode('utf-8') + '\033[0m')
+    logger.log(u'\033[92m' + message.decode('utf-8') + '\033[0m')
 
 
 def print_yellow(message):
-    print(u'\033[93m' + message.decode('utf-8') + '\033[0m')
+    logger.log(u'\033[93m' + message.decode('utf-8') + '\033[0m')
 
 
 def print_red(message):
-    print(u'\033[91m' + message.decode('utf-8') + '\033[0m')
+    logger.log(u'\033[91m' + message.decode('utf-8') + '\033[0m')
```

Comment:
Are these three functions being used at all anywhere else? If not, we can probably delete them too.